CommonLib/Error/GREYError.m

```diff
@@ -108,6 +110,11 @@ + (instancetype)errorWithDomain:(NSString *)domain
 }
 
 - (NSString *)description {
+  if (self.domain == kGREYInteractionErrorDomain &&
```

Comment:
Move to a static function in GREYErrorFormatter. You could even just have this as checked in formattedDescription.

NSString *des = [[[GREYErrorFormatter alloc] initWithError:self] formattedDescription];
if (!des) {
  return [GREYObjectFormatter formatDictionary:[self grey_descriptionDictionary]
                                          indent:kGREYObjectFormatIndent
                                       hideEmpty:YES...
}